src/search/components/query-editor.controller.js

```diff
@@ -193,6 +185,15 @@ function QueryEditorController(
       return node.nodes.length > 0;
     });
 
+    if (groupedTree.nodes[0].operator === 'AND' &&
+      groupedTree.nodes.length > 1 &&
+      groupedTree.nodes[0].nodes.length === 1
+    ) {
+      var firstGroup = groupedTree.nodes[0];
```

Comment:
You can declare this variable above the if and reuse it in the if statement

Reply:
What's the benefit of doing this?
It's only necessary if the condition is true?

Reply:
Ah I misread the code. Never mind